articles/databox-online/azure-stack-edge-gpu-clustering-overview.md

```diff
@@ -73,70 +73,69 @@ The following network topologies are available:
-    
-    - **With Port 1 and Port 2 in same subnets** - In this case, Port 1 and Port 2 have a teamed virtual switch and both the ports are in the same subnet.
-
-    In each case, Port 3 and Port 4 are connected back-to-back directly without a switch. These ports are dedicated to storage and Azure Stack Edge cluster traffic and aren't available for workload traffic. 
+- **Option 1** - Port 1 and Port 2 are in different subnets. Separate virtual switches will be created. Port 3 and Port 4 connect to an external virtual switch.
+ 
+- **Option 2** - Port 1 and Port 2 are in the same subnet. A teamed virtual switch will be created. Port 3 and Port 4 connect to an external virtual switch.
 
+- **Option 3** - Port 1 and Port 2 are in separate subnets. A teamed virtual switch will be created. Port 3 and Port 4 are connected back-to-back, switchless for Port 3 and Port 4.
 
-- **Using external switches** - Use this option when you have high speed switches (10 GbE switches) available for use with your device nodes for storage and cluster traffic. There are further sub-options:
+- **Option 4** - Port 1 and Port 2 are in the same subnet. A teamed virtual switch will be created. Port 3 and Port 4 are connected back-to-back, switchless for Port 3 and Port 4.
 
-    - **With Port 1 and Port 2 in separate subnets** - This is the default option. In this case, Port 1 and Port 2 have separate virtual switches and are connected to separate subnets.
-    
-    - **With Port 1 and Port 2 in same subnets** - In this case, Port 1 and Port 2 have a teamed virtual switch and both the ports are in the same subnet.
+  > [!NOTE]
+  > If you run PMEC workloads, use Option 1 or Option 2.
 
-    In each case, Port 3 and Port 4 are reserved for Private Multi-Access Edge Computing workload deployments. 
+Usage considerations on your Azure Stack Edge Pro 2 device nodes:
 
-The pros and cons for each of the above supported topologies can be summarized as follows:
+- **Switchless for Port 3 and Port 4** - Use this option when you don't have high speed switches available in the environment, or you want to dedicate Port 3 and Port 4 for storage and cluster traffic.
+  - **Port 1 and Port 2 in separate subnets** - This is the default option. In this case, Port 1 and Port 2 have separate virtual switches and are connected to separate subnets.
+  - **Port 1 and Port 2 in the same subnet** - In this case, Port 1 and Port 2 have a teamed virtual switch and both ports are in the same subnet.
+- **Using external switches for Port 3 and Port 4** - Use this option when you have high speed switches (>=10 GbE bandwidth) available for use with your device nodes and you want to allow a VM network adapter to connect to the virtual network created on Port 3 or Port 4, like a PMEC use case.
+- Additional options for **Switchless** scenarios that use **external switches**:
```

Comment:
Line 111 can be removed, as line 112 and 113 are subsection of line 110.

Reply:
Done > updated the PR to remove line 111. Thank you.